packages/typed-contracts/abi/EtherToken.json

```diff
@@ -0,0 +1,233 @@
+[
```

Comment:
Hm... Do we want to commit these and have them in two places or should we add it to `gitignore` and have an `npm run` command the copies them over from the `contracts` subpackage?